<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhyllotaxisEditor</class>
 <widget class="QDialog" name="PhyllotaxisEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>416</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>81</width>
     <height>241</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="CurveWidget" name="curveWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>261</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="expressionPlainTextEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
  </widget>
  <widget class="QLabel" name="expressionLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Density Function</string>
   </property>
  </widget>
  <widget class="QLabel" name="yMinLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="yMaxLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>1.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="xMinLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="xMaxLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>230</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CurveWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>Widgets/CurveWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PhyllotaxisEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PhyllotaxisEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
